pkg/storkctl/migration.go

```diff
@@ -251,6 +271,29 @@ func newDeactivateMigrationsCommand(cmdFactory Factory, ioStreams genericcliopti
 				updateCronJobObjects(ns, false, ioStreams)
 			}
 
+			for _, ns := range migrationScheduleNamespaces {
+				migrSched, err := storkops.Instance().ListMigrationSchedules(ns)
+				if err != nil {
+					util.CheckErr(err)
+					return
+				}
+				for _, migr := range migrSched.Items {
+					if migr.GetAnnotations() != nil {
+						if _, ok := migr.GetAnnotations()[StorkMigrationScheduleCopied]; ok {
+							// check status of all migrated app in cluster
+							printMsg(fmt.Sprintf("Setting AppActive status to false in remote migration schedule for %v/%v ", migr.Namespace, migr.Name), ioStreams.Out)
+							migr.Status.ApplicationActivated = false
+							_, err := storkops.Instance().UpdateMigrationSchedule(&migr)
+							if err != nil {
+								util.CheckErr(err)
+								return
+							}
+							printMsg(fmt.Sprintf("Updated migrationschedule AppActive status for %v/%v to false successfully", migr.Namespace, migr.Name), ioStreams.Out)
```

Comment:
Lets keep the messaging for activation and deactivation the same. Can you also fix the message on line 202
```suggestion
printMsg(fmt.Sprintf("Set the ApplicationActivated status in the MigrationSchedule %v/%v to false", migr.Namespace, migrSched.Name), ioStreams.Out)
```